### DISCUSSION TITLE
제페토 빌드잇과 유니티

### DISCUSSION THREAD

**Discussion by kkkja** [naverz/zepeto-studio-kor#981] [5. Q&A]
협업 프로젝트를 시작하게 되었습니다.

1.  빌드잇에서 작업한 월드를 유니티로 불러서 작업이 가능할까요..?

2.  유니티에서는 서로 각각 월드 간 이동할 수 있는 포털 연결이 가능할까요..?

3.  상호작용을 통해서 화면에 팝업창을 띄우는 것이 가능할까요?

**Comment by pumfit** [comment]
1. 가능합니다! 해당 가이드 참고하시면 됩니다. https://studio.zepeto.me/guides/buildit 
2. 월드간 이동은 아직 지원되지 않는 것 같습니다 https://github.com/naverz/zepeto-studio-kor/discussions/607
3. 상호작용도 가능합니다 UI 관련 가이드는 여기를 참고하시면 될 것 같습니다. https://studio.zepeto.me/kr/guides/ui-events
프로젝트 진행하시기 전에 가이드 월드 시작하기 부분 참고하면 도움 되실 것 같아요!
